Source/TrackableEntities.EF.5/DbContextExtensions.cs

```diff
@@ -225,7 +234,7 @@ public static void ApplyChanges(this InterceptorPool pool, IEnumerable<ITrackabl
                 else
                 {
                     context.ApplyChangesOnProperties(item, visitationHelper.Clone(), TrackingState.Modified, interceptors); // Clone to avoid interference
-                    context.ApplyChangesOnProperties(item, visitationHelper, TrackingState.Deleted, interceptors); 
```

Comment:
@weitzhandler Make sure you don't commit changes to things like spacing and indentation, so those files won't appear in the diff.  That will help me focus on the changes that really matter. :)